source/index.html.md

```diff
@@ -161,35 +164,37 @@ video_title | example: 'Awesome Show Episode 1'
 video_series | example: 'Season 1'
 video_variant_name | An optional detail that allows you to monitor issues with the files of specific versions of the content, for example different audio translations or versions with hard-coded/burned-in subtitles.
 video_variant_id | Your internal ID for a video variant
-video_language | The audio language of the video, assuming it's unchangeable after playing.
+video_language_code | The audio language of the video, assuming it's unchangeable after playing.
 video_content_type | 'short', 'movie', 'episode', 'clip', 'trailer', or 'event'
-video_duration | The length of the video in **seconds** [float]
+video_duration | The length of the video in **milliseconds** [integer]
 video_stream_type | 'live' or 'on-demand'
 video_producer | The producer of the video title
 video_encoding_variant | An optional detail that allows you to compare different encoding settings.
+video_cdn | An optional detail that allows you to compare different CDNs (assuming the CDN selection is made at page load time).
 
 ## Changing the Video
 
 ```video-element
 // Example of changing the source of a video element
-// Should happen before changeVideo is called
+// Should happen before emitting the videochange event
 var myVideo = document.querySelector('#my-video');
 myVideo.src = 'nextVideo.mp4';
 
-mux.monitor('#my-video').changeVideo({
+mux.emit('#my-video', 'videochange', {
   video_id: 'abc345',
   video_title: 'My Other Great Video',
-  video_series: 'Weekly Great Videos'
+  video_series: 'Weekly Great Videos',
+  ...
```

Comment:
That's a good point I hadn't thought about, actually. Personally I would rather see the ... and have it break than forget that I'm supposed to put things in there, but I could be convinced to take them out.